src/style.css

```diff
@@ -0,0 +1,119 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.product-card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+}
+
+.product-card:hover {
+  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.07);
+}
+
+.product-image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin-bottom: 15px;
+  padding: 30px 19px;
+}
+
+.product-info {
+  padding: 0 14px 14px;
+}
+
+.product-name {
+  margin: 0 0 6px;
+  color: #000;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 1.25em;
+}
+
+.product-sku {
+  margin: 0 0 15px;
+}
+
+.product-sku,
+.product-discussion,
+.price {
+  font-size: 11px;
+  font-weight: 400;
+  line-height: 1.18em;
+}
+
+.product-rating-discussion {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 20px;
+}
+
+.product-rating {
+  display: flex;
+}
+
+.rating__star {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 15px;
+  height: 15px;
+}
+
+.rating__star img {
+  max-width: 15px;
+  max-height: 15px;
+}
+
+.rating__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.product-discussion {
+  margin: 0;
+  padding: 2px 0 0;
+}
+
+.product-price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 20px;
+}
+
+.price {
+  align-self: center;
+}
+
+.price-value {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 1.19em;
+}
+
+.btn {
+  display: inline-block;
+  width: 100%;
+  height: 38px;
+  background-color: #00acdc;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+  color: #fff;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 38px;
```

Comment:
I'm not sure you need it here.

Reply:
line-height: 38px; - this is a vertical align for button's title